app/src/pages/inside/logsPage/logToolbar/logToolbar.jsx

```diff
@@ -149,10 +147,11 @@ export class LogToolbar extends Component {
     return fetchTestItems({ next: true });
   };
 
-  changeTypeOfHistory = (type) => {
-    const { tracking } = this.props;
+  changeHistoryLineMode = () => {
+    const { tracking, includeAllLaunches } = this.props;
     tracking.trackEvent(LOG_PAGE_EVENTS.HISTORY_LINE_MODE_CHB);
-    this.props.fetchLineHistoryAction(type);
+    this.props.setIncludeAllLaunches(!includeAllLaunches);
+    this.props.fetchLineHistoryAction();
```

Comment:
Not need to dispatch another action in this case, action from `setIncludeAllLaunches` action creator may be directly catched by sagas.